integration/integration.go

```diff
@@ -61,49 +63,91 @@ type ServerOption interface {
 	applyServerOption(*serverOptions)
 }
 
+// TODO: What does this do?
```

Comment:
See: https://github.com/pulumi/pulumi-go-provider/pull/363/files#diff-a3d303ad5b0de361788b41ef8df6b57b4ddb2fb21e841b4b2b1dbf2f67c83a3dR19

Reply:
Ahhhh thank you!